std/file.d

```diff
@@ -852,14 +859,17 @@ unittest
     Throws:
         $(D FileException) on error.
  +/
-version(StdDdoc) void getTimesWin(in char[] name,
+version(StdDdoc) void getTimesWin(R)(R name,
```

Comment:
Which is why for many of them I split out the implementation code into a separate function from the code that makes the 0 terminated argument.